app/services/analytics_events.rb

```diff
@@ -5179,6 +5179,31 @@ def idv_warning_shown(
     )
   end
 
+  # @param [Array] error_details Full messages of the errors
+  # @param [Hash] error_types Types of errors that are surfaced
+  # @param [Symbol] event What part of the workflow the error occured in
+  # @param [Boolean] integration_exists Whether the requesting issuer maps to an SP
+  # @param [String] request_issuer The issuer in the request
+  def integration_errors_present(
```

Comment:
suggestion: Consider prefixing the method with something like `sp_` or `idp_`. As we have integrations with other vendors as well as registered service_provider integrations, this current name is a bit confusing without a lot more context.

Reply:
updated in https://github.com/18F/identity-idp/pull/11615/commits/d011de2a5217dddf83ecedde0a086c8efb1f7eea